.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -43,6 +43,7 @@ Providing this information upfront will facilitate a smoother review process. --
 
 ## Checklist
 - [ ] Please read our [Vector contributor resources](https://github.com/vectordotdev/vector/tree/master/docs#getting-started).
+  - Ensure the following commands are passing locally: `cargo fmt`, `cargo clippy --all` and `cargo test --all`.
```

Comment:
I think the flags we pass to `clippy` and `test` differ in CI, so there may still be some things missed, but otherwise 👍

Reply:
You could also consider pointing people to `cargo vdev ...`. Like `cargo vdev check rust` and `cargo vdev check fmt`.

Reply:
Maybe we could even add something like `cargo vdev check all` that runs them all 🤔